test/js/ez-selection-activate-element.js

```diff
@@ -0,0 +1,115 @@
+describe('ez-selection-activate-element', function() {
+    let element;
+    let elementSingleCheck;
+    let elementSubsetElementChecked;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+        elementSingleCheck =  fixture('TestFixtureSingleElementChecked');
+        elementSubsetElementChecked =  fixture('TestFixtureSubsetElementChecked');
+    });
+
+    function checkAndclickOneElement(elem, selector, isChecked) {
+        const checkbox = elem.querySelector(selector);
+
+        checkbox.checked = isChecked;
+        checkbox.dispatchEvent(new CustomEvent('click', {
+            bubbles: true,
+        }));
+    }
+
+    function checkAndclickEveryElements(elem, selector, isChecked) {
+        const checkboxes = elem.querySelectorAll(selector);
+
+        checkboxes.forEach((checkbox) => {
+            checkbox.checked = isChecked;
+            checkbox.dispatchEvent(new CustomEvent('click', {
+                bubbles: true,
+            }));
+        });
+    }
+
+    it('should be defined', function () {
+        assert.equal(
+            window.customElements.get('ez-selection-activate-element'),
+            element.constructor
+        );
+    });
+
+    describe('properties', function () {
+        describe('`selectionSelector`', function () {
+            it('should default to `input[type="checkbox"]``', function () {
+                assert.equal(element.selectionSelector, 'input[type="checkbox"]');
+            });
+
+            describe('set', function () {
+                it('should be reflected to an attribute', function () {
+                    element.selectionSelector = '.someClass';
+
+                    assert.equal(
+                        element.getAttribute('selection-selector'),
+                        element.selectionSelector
+                    );
+                });
+            });
+        });
+    });
+
+    describe('activate element', function () {
```

Comment:
That seems quite complicated, I think the outline of the tests here should be something like:

```js
describe('activate elements', function () {
    describe('single selection', function () {
        it('should enable activable buttons');
    });

    describe('multiple selection', function () {
        it('should enable activable button');

        it('should disable button bound to a single selection');
    })

    describe('no selection', function () {
        it('should disable elements');
    });
});
```

where you can use only one fixture containing 2 activable buttons (one bound to a single selection).